test/fixtures/patients.yml

```diff
@@ -429,7 +429,7 @@ patient_15:
   first_name: 'Jon'
   last_name: 'Snow'
   sex: 'Unknown'
-  date_of_birth: <%= 50.years.ago %>
+  date_of_birth: <%= 51.years.ago %>
```

Comment:
This is the fix for the AnalyticsJobTest failures. These were failing because the Rails date and MySQL CURDATE() disagreed and this patient got put into the 40-49 age group bucket instead of the 50-59 age group bucket

Reply:
Just to close the loop on this - locally MySQL is usually running with SYSTEM time zone and the Rails `years.ago` methods convert to UTC which will be the next day.